rfcs/accessibility-checklist.md

```diff
@@ -0,0 +1,52 @@
+# Blade Component Accessibility Checklist <!-- omit in toc -->
+
+This document will contain information and guidance on how to ensure accessibility of components,
+And how we can incorporate it in the component development life cycle.
+
+## 5 Minute Guide for ensuring accessibility in blade components
+
+https://user-images.githubusercontent.com/35374649/175493627-4c5f29e9-0625-44fe-a5a8-03df7700f886.mp4
+
+## Checklist
+- [ ] Add `Accessibility` section to component's `decision.md`. [More Details](#component-decisions)
+- [ ] Use `makeAccessible` utility to make components accessible across platforms. [More Details](#implementations)
+- [ ] Adhere to the aria states & roles for the component
```

Comment:
can we add w3 link for this item?

Reply:
Not sure which link should I put. These links will be component specific 🤔 not anything general.

Reply:
Okay.. added (Example) links.